spec/models/concerns/user/soft_delete_concern_spec.rb

```diff
@@ -0,0 +1,183 @@
+RSpec.describe User::SoftDeleteConcern do
+  describe "#soft_delete!" do
+    it "change email to a « deleted.rdv-solidarites.fr » domain and anonymises other attributes" do
```

Comment:
Est-ce qu’on peut mettre tous les it en FR ? (ou en tout cas harmoniser parce qu’il y a de l’anglais au début puis du français)

Reply:
à quand le `alias :ça :it` ? 🥖 🇫🇷